.github/workflows/ci.yml

```
@@ -61,3 +61,38 @@ jobs:
     - name: Documentation
       run: |
         cabal haddock
+
+  stack:
+      name: stack / ghc ${{ matrix.ghc }}
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          stack: ["2.7.5"]
+          ghc: ["9.2.3"]
+
+      steps:
+      - uses: actions/checkout@v2
+
+      - uses: haskell/actions/setup@v1.2
```

Let's use the same version of Haskell action as for cabal jobs
```suggestion
      - uses: haskell/actions/setup@v2.0
```